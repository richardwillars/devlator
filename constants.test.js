const constants = require("./constants");

describe("Constants", () => {
  it("should export all the constants", () => {
    expect(constants).toMatchSnapshot();
  });
});
