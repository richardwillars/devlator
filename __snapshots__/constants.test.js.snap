// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Constants should export all the constants 1`] = `
Object {
  "DEVICE_EVENT": "DEVICE:EVENT",
  "DEVICE_LIGHT_EVENT": "DEVICE_LIGHT:EVENT",
  "DEVICE_SENSOR_EVENT": "DEVICE_SENSOR:EVENT",
  "DEVICE_SOCKET_EVENT": "DEVICE_SOCKET:EVENT",
  "DEVICE_SPEAKER_EVENT": "DEVICE_SPEAKER:EVENT",
  "events": Object {
    "ADDED_TO_QUEUE_BOTTOM": "ADDED_TO_QUEUE_BOTTOM",
    "ADDED_TO_QUEUE_NEXT": "ADDED_TO_QUEUE_NEXT",
    "AUDIO_PLAYING_STATE": "AUDIO_PLAYING_STATE",
    "BATTERY_LEVEL": "BATTERY_LEVEL",
    "BREATHE_LIGHT_EFFECT": "BREATHE_LIGHT_EFFECT",
    "CONTACT": "CONTACT",
    "CURRENT_AUDIO_TRACK": "CURRENT_AUDIO_TRACK",
    "ENERGY": "ENERGY",
    "HUMIDITY": "HUMIDITY",
    "LED_STATE": "LED_STATE",
    "LIGHT": "LIGHT",
    "LIGHT_STATE": "LIGHT_STATE",
    "MOTION": "MOTION",
    "MUTED_AUDIO": "MUTED_AUDIO",
    "NAME": "NAME",
    "NEXT_AUDIO_TRACK": "NEXT_AUDIO_TRACK",
    "ON": "ON",
    "PREVIOUS_AUDIO_TRACK": "PREVIOUS_AUDIO_TRACK",
    "PULSE_LIGHT_EFFECT": "PULSE_LIGHT_EFFECT",
    "QUEUE_FLUSHED": "QUEUE_FLUSHED",
    "SEEK": "SEEK",
    "TAMPER": "TAMPER",
    "TEMPERATURE": "TEMPERATURE",
    "UV": "UV",
    "VIBRATION": "VIBRATION",
    "VOLUME": "VOLUME",
  },
  "sensor": Object {
    "BATTERY_LEVEL": "BATTERY_LEVEL",
    "CONTACT": "CONTACT",
    "HUMIDITY": "HUMIDITY",
    "LIGHT": "LIGHT",
    "MOTION": "MOTION",
    "NAME": "NAME",
    "TAMPER": "TAMPER",
    "TEMPERATURE": "TEMPERATURE",
    "UV": "UV",
    "VIBRATION": "VIBRATION",
  },
  "socket": Object {
    "ENERGY": "ENERGY",
    "NAME": "NAME",
    "ON": "ON",
  },
  "speaker": Object {
    "ADDED_TO_QUEUE_BOTTOM": "ADDED_TO_QUEUE_BOTTOM",
    "ADDED_TO_QUEUE_NEXT": "ADDED_TO_QUEUE_NEXT",
    "AUDIO_PLAYING_STATE": "AUDIO_PLAYING_STATE",
    "CURRENT_AUDIO_TRACK": "CURRENT_AUDIO_TRACK",
    "LED_STATE": "LED_STATE",
    "MUTED_AUDIO": "MUTED_AUDIO",
    "NAME": "NAME",
    "NEXT_AUDIO_TRACK": "NEXT_AUDIO_TRACK",
    "PREVIOUS_AUDIO_TRACK": "PREVIOUS_AUDIO_TRACK",
    "QUEUE_FLUSHED": "QUEUE_FLUSHED",
    "SEEK": "SEEK",
    "VOLUME": "VOLUME",
  },
}
`;
